content/en/istio-ca-certs-integrations/cert-manager-integration/_index.md

```diff
@@ -141,39 +138,38 @@ spec:
           name: istiod
           patches:
             # Mount istiod serving and webhook certificate from Secret mount
-          - path: spec.template.spec.containers.[name:discovery].args[7]
+          - path: spec.template.spec.containers.[name:discovery].args[-1]
             value: "--tlsCertFile=/etc/cert-manager/tls/tls.crt"
-          - path: spec.template.spec.containers.[name:discovery].args[8]
+          - path: spec.template.spec.containers.[name:discovery].args[-1]
             value: "--tlsKeyFile=/etc/cert-manager/tls/tls.key"
-          - path: spec.template.spec.containers.[name:discovery].args[9]
+          - path: spec.template.spec.containers.[name:discovery].args[-1]
             value: "--caCertFile=/etc/cert-manager/ca/root-cert.pem"
-          - path: spec.template.spec.containers.[name:discovery].volumeMounts[6]
+          - path: spec.template.spec.containers.[name:discovery].volumeMounts[-1]
             value:
               name: cert-manager
               mountPath: "/etc/cert-manager/tls"
               readOnly: true
-          - path: spec.template.spec.containers.[name:discovery].volumeMounts[7]
+          - path: spec.template.spec.containers.[name:discovery].volumeMounts[-1]
             value:
               name: ca-root-cert
               mountPath: "/etc/cert-manager/ca"
               readOnly: true
-          - path: spec.template.spec.volumes[6]
+          - path: spec.template.spec.volumes[-1]
             value:
               name: cert-manager
               secret:
                 secretName: istiod-tls
-          - path: spec.template.spec.volumes[7]
+          - path: spec.template.spec.volumes[-1]
             value:
               name: ca-root-cert
               configMap:
-                secretName: istiod-tls
                 defaultMode: 420
                 name: istio-ca-root-cert
 EOF
-getmesh istioctl install -f config.yaml
 ```
 
-Note: if you are using Istio 1.7.x, make sure to update the `args` under the `patches` filed in the above YAML as follows:
+__Note:__ If you are using Istio 1.7.x, make sure to update the `args` under the `patches`
```

Comment:
Feel we don't need to support 1.7? Neither Istio or us are using that, right? This could simplify the doc, by making it shorter.

Reply:
I removed this, it was part of the original doc.